lmnet/lmnet/networks/quantize_param_init.py

```diff
@@ -0,0 +1,123 @@
+import tensorflow as tf
+import functools
+
+
+class QuantizeParamInit:
+    """QuantizeParamInit is a mixin creating a quantized network class more convenient by providing two functions.
+    1) __init___ : for initializing variables necessary for quantization, such as quantization function
```

Comment:
These functions `__init___` and `_quantized_variable_getter` is not public variable or function. 🤔🤔🤔 
Please write about public var and func.

Reply:
`_quantized_variable_getter` is a private function so I will delete the explanation about it.

but I think `__init__` is a public function called via the network class. (e.g. when a network object is generated)
Should I delete explanation about `__init__`?

Reply:
All of python class have `__init__` , don't need to write describe about what is `__init__`,  I think :+1: